src/calls-cost.js

```diff
@@ -20,6 +20,32 @@
  * @property {ProcessedCall[]} callsDetails - Processed information about calls
  */
 
+function calculate(duration, additionalCost, firstminutes = 0, cost = 0) {
```

Comment:
I think `firstMinutes` in camel case keeps the standard you used with `additionalCost`.